docs/csharp/language-reference/configure-language-version.md

```diff
@@ -20,6 +20,17 @@ You can override the default behavior by specifying a language version. There ar
 - Set the language version [for multiple projects in a subdirectory](#configure-multiple-projects).
 - Configure the [`-langversion` compiler option](#set-the-langversion-compiler-option).
 
+## Required target framework
+
+Language features may rely on library types or APIs in a target framework. Therefore, in order to use a given language version, you must select a target framework that has the required APIs. The following table lists the *minimum* required target framework for recent language versions. You may target any higher version as well.
+
+- **C# 8.0**: *.NET Standard 2.1*, including *.NET Core 3.0*.
+- **C# 7.3**: *.NET Standard 2.0*, including *.NET Core 2.1*, and *.NET Framework 4.7*.
+- **C# 7.2**: *.NET Standard 2.0*, including *.NET Core 2.1*, and *.NET Framework 4.7*.
+- **C# 7.1**: *.NET Standard 2.0*, including *.NET Core 2.1*, and *.NET Framework 4.7*.
+- **C# 7.0**: *.NET Standard 2.0*, including *.NET Core 2.1*, and *.NET Framework 4.7*.
```

Comment:
How does this work exactly? Because I can target `<LangVersion>8.0</LangVersion>` and `netcoreapp2.0` without issue.

Reply:
@Thraka You can, and you'll be able to use some C# 8.0 features, but not all of them (e.g. async streams require .Net Core 3.0).

I guess the text could make that clearer, something like:

> Therefore, in order to use <ins>all features in</ins> a given language version, you must select a target framework that has the required APIs.